packages/babel-traverse/src/path/context.js

```diff
@@ -92,12 +93,14 @@ export function skip() {
 }
 
 export function skipKey(key) {
+  if (this.skipKeys == null) {
+    this.skipKeys = {};
+  }
```

Comment:
I wish https://github.com/tc39/proposal-logical-assignment wasn't just stage 1 :stuck_out_tongue:

Reply:
Offtopic: we still need a PR to add nullish coalescing and optional chaining.

Reply:
We can always use new stage features if you are willing to move off of them if they change (I think that one is small enough that it's fine)